jenkins-jobs/nci/project.rb

```diff
@@ -19,15 +19,22 @@
 # License along with this library.  If not, see <http://www.gnu.org/licenses/>.
 
 require_relative '../../ci-tooling/lib/nci'
-require_relative '../builder'
+require_relative '../sourcer'
+require_relative '../binarier'
+require_relative '../publisher'
 require_relative 'multijob_phase'
 
 # Magic builder to create an array of build steps
 class ProjectJob < JenkinsJob
   def self.job(*args, **kwords)
```

Comment:
This should now be expanded to actual vars instead of messing about with kwords and args manually `(project, distribution:, architectures:, type:)` if I am not mistaken.
The expansion of variables below can be dropped and all access of `kwords[]` should be replced by the actual variables.